docs/userguide/autoscaling/federatedhpa.md

```diff
@@ -35,3 +35,4 @@ If you configure FederatedHPA, you may also want to consider running a cluster-l
 
 For more information on FederatedHPA:
 * Read a [FederatedHPA tutorials](../../tutorials/autoscaling-with-federatedhpa.md) for FederatedHPA.
```

Comment:
How about renaming this to `FederatedHPA tutorials with CPU/Memory utilization metrics`?

Reply:
Sorry, I mean changing line 37 to this.